h/views/api_groups.py

```diff
@@ -50,3 +77,16 @@ def remove_member(group, request):
     group_service.member_leave(group, userid)
 
     return HTTPNoContent()
+
+
+# @TODO This is a duplication of code in h.views.api — move to a util module
```

Comment:
This code is a blatant copy and paste job from https://github.com/hypothesis/h/blob/5ec4d0b11aeb54deec19a19f041999448d15f975/h/views/api.py#L211

I have a TODO to put this utility function somewhere reusable

Reply:
Maybe `request.parsed_json_body`? (Lets get rid of the word "payload" being used for what both Pyramid and HTTP call the "body" too)